bin/ch-test

```diff
@@ -71,6 +71,10 @@ builder_check () {
         bl=$(command -v docker)
         bv=$(docker_ --version | awk '{print $3}' | sed -e 's/,$//')
         ;;
+    none)
+        bl=
+        bv= 
```

Comment:
An issue I have with this is the test suite output:

```
checking namespaces ...
ok

checking builder ...
found:

creating /var/tmp/tar
[...]
```

I'd prefer an explicit output here instead implicit.

Reply:
Yes, I think you're right.